src/App/Datums.cpp

```diff
@@ -401,3 +387,11 @@ bool LocalCoordinateSystem::LCSExtension::extensionGetSubObject(DocumentObject*&
         return false;
     }
 }
+
+bool LocalCoordinateSystem::hasObject(const DocumentObject* obj, bool recursive) const
+{
+    Q_UNUSED(recursive);
```

Comment:
```suggestion
bool LocalCoordinateSystem::hasObject(const DocumentObject* obj, [[maybe_unused]] bool recursive) const
{
```

Reply:
Is there a reason to favor [[maybe_unused]] instead of Q_UNUSED? ie should I from now on use [[maybe_unused]] ?

Reply:
Ah yes that makes sense. No point adding more dependency to qt than necessary indeed

Reply:
But in Gui is it also better to use [[maybe_unused]] then?